pkgs/development/libraries/p11-kit/default.nix

```diff
@@ -63,6 +63,13 @@ stdenv.mkDerivation rec {
     ]))
   ];
 
+  # Tests run in fakeroot for non-root users
+  preCheck = ''
+    if [ "$(id -u)" != "0" ]; then
```

Comment:
Does this break things if we set this variable always? Otherwise people on just nixos cannot know, if it continue to works on other platforms.

Alsop lease follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) when naming your commits.

Reply:
Thanks for such a speedy review!

> Alsop lease follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions) when naming your commits.

Apologies, I believe it now follows the convention!

> Does this break things if we set this variable always? Otherwise people on just nixos cannot know, if it continue to works on other platforms.

I haven't thought about this change too much, I'm just reverting it back to how it was before https://github.com/NixOS/nixpkgs/commit/3ca33e5ef4f8c3e24013905cf741330c9ace3a6c#diff-6e5cbf6c584d88a4b8326fc03daf8650866eed33b1651b38ab379da599f57109L72-L77.

Reply:
It would skip the tests from running when building on NixOS/multi-user as well.